src/pybind/mgr/dashboard/frontend/src/app/shared/validators/cd-validators.ts

```diff
@@ -49,17 +40,20 @@ export class CdValidators {
    *   argument. The function must return true to set the validation error.
    * @return {ValidatorFn} Returns the validator function.
    */
-  static requiredIf(prerequisites: Prerequisites, condition?: Function | undefined): ValidatorFn {
+  static requiredIf(prerequisites: Object, condition?: Function | undefined): ValidatorFn {
     return (control: AbstractControl): ValidationErrors | null => {
       // Check if all prerequisites matches.
-      if (!Object.keys(prerequisites).every((key) => {
-        return (control.parent && control.parent.get(key).value === prerequisites[key]);
-      })) {
+      if (
+        !Object.keys(prerequisites).every((key) => {
+          return control.parent && control.parent.get(key).value === prerequisites[key];
+        })
+      ) {
         return null;
       }
-      const success = _.isFunction(condition) ? condition.call(condition, control.value) :
-        isEmptyInputValue(control.value);
-      return success ? {'required': true} : null;
+      const success = _.isFunction(condition)
+        ? condition.call(condition, control.value)
```

Comment:
Don't like the operator at the beginning of the line - it should be at the end of the last line to be more clear about that the line is continuing.

```
question ?
yes :
no
```

Reply:
ATM you can't change how this is formatted. 
There is an open discussion over at prettier repo regarding this: https://github.com/prettier/prettier/issues/737 .

I don't mind this, with this you can look at 3 consecutive long lines and figure out what each does.

Regarding the end of the line, it's the same situation as when you have multiple piped methods, you know its continuing when there is no `;` on the end of the line and the next method is on a new line.

Reply:
The issue mentioned is not the right issue - I mend prettier/prettier#716 ,
but reading this I'm OK with it.